src/hyperfine/export/mod.rs

```diff
@@ -51,16 +54,19 @@ impl ExportEntry {
 
 /// The ResultExportType enum is used to denote the desired form
 /// of exporter to use for a given file.
+#[derive(Clone)]
 pub enum ResultExportType {
```

Comment:
Maybe consider using this `enum` just to track the type of export and pass the file path as a separate argument? Just a thought.

Reply:
Well, as I was looking at it, these two pieces of data are intrinsically linked to one-another. I didn't really see any benefit in breaking them apart, just to have to bring them back together or store the file name somewhere else. However, I just noticed that there is no reason to derive clone on this type and it can safely be removed.

Reply:
See #46